### ISSUE TITLE
[CREATE_DB] not able to type in password

### ISSUE THREAD

**Issue by Ferkelcode** [3dcitydb/3dcitydb#84]
Hello,

After providing SRID, EPSG and srsName, I was asked to provide password, however I am not able to type in. Do you know why?

**Comment by clausnagel** [comment]
The password input is hidden by intention. Just type in your password and press enter to continue.

**Comment by Ferkelcode** [comment]
> The password input is hidden by intention. Just type in your password and press enter to continue.

Thanks a lot, I have typed in the password and press enter but it shows:

> psql: Fehler: Verbindung zum Server auf »localhost« (::1), Port 5432 fehlgeschlagen: FATAL:  Passwort-Authentifizierung für Benutzer »mydb« fehlgeschlagen

The password I typed in is correct, does this mean that I need to be a superuser?

**Comment by clausnagel** [comment]
You do not need superuser privileges to setup a 3DCityDB (see [here](https://3dcitydb-docs.readthedocs.io/en/version-2022.0/first-steps/setup-3dcitydb.html#installation-steps-on-postgresql).

The password authentication for your user "mydb" failed. This is not a 3DCityDB issue. Make sure that you have set up the user correctly. For example, check whether you can login with the user using a tool like `pgAdmin` or on the command-line using `psql`. In fact, the 3DCityDB setup scripts also uses `psql` in the background.

**Comment by Ferkelcode** [comment]
Thanks a lot.

With this password, I am able to login pgAdmin, but here with the same password, I always get the same error.

________________________________

**Comment by Ferkelcode** [comment]
> You do not need superuser privileges to setup a 3DCityDB (see [here](https://3dcitydb-docs.readthedocs.io/en/version-2022.0/first-steps/setup-3dcitydb.html#installation-steps-on-postgresql)).
> 
> The password authentication for your user "mydb" failed. This is not a 3DCityDB issue. Make sure that you have set up the user correctly. For example, check whether you can login with the user using a tool like `pgAdmin` or on the command-line using `psql`. In fact, the 3DCityDB setup scripts also uses `psql` in the background.

Thanks a lot. 

With this password, I am able to login pgAdmin, but here with the same password, I always get the same error.

**Comment by Ferkelcode** [comment]
> Make sure that you have set up the user correctly. For example, check whether you can login with the user using a tool like `pgAdmin` or on the command-line using `psql`. In fact, the 3DCityDB setup scripts also uses `psql` in the background.

Thank you for the answer. 

Since I cannot see what I typed in for the password, is that possible that there is no input even though I typed. As I have typed in ( I have also try to copy & paste) the password for many times, I think the password input is not wrong. And I have the access to pgAdmin. So I was wondering if the reason can be that there is no input to the password?

Thanks a lot.
